### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by morganjacobus**
This is the in-progress draft for the README. I still need to add how to pull the files down from gitHub and run as well as descriptions of the folders and the files/data within.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+## What is the purpose of this repository?
+The purpose of this repository is to house the data, code, figures, and products for the Bus Stop Census data anaylsis. MARTA Army has been collecting data on MARTA bus stops from February 4th, 2020 to Dec 31st, 2020. With data collection over, the task remains to clean, validate, and analyze the data. The repository documents this process and provides an organized and replicable framework for the public to use for their own analyses.
+
+## R and RStudio Set Up Instructions
+To get started, follow these instructions to install R and RStudio adapted from [Copyright (c) Software Carpentry](http://software-carpentry.org).
+
+### R and RStudio
+- Download and install [R, a free software environment for statistical computing and graphics](https://www.r-project.org/) from [CRAN](https://cran.rstudio.com/), the Comprehensive R Archive Network. It is highly recommended to install a precompiled binary distribution for your operating system – use the links up at the top of the CRAN page linked to above!
```
Comment (QYAN24) [inline README.md]: Let's use only one of these links. Which one do you prefer?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+- Install RStudio's IDE (stands for integrated development environment), a powerful user interface for R: http://www.rstudio.com/ide/download/
+  - RStudio includes a text editor, so you do not have to install another stand-alone editor.
+  - RStudio includes an interface to Git and GitHub. You will still need to install Git (covered below) but RStudio provides a basic GUI for interacting with Git(Hub).
+  
+- Install [Git](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/set-up-git)
+  - To access this repository, you will need to create an account on [GitHub.com](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/signing-up-for-a-new-github-account).
```
Comment (QYAN24) [inline README.md]: Not needed. IMO having git is implied, and a GitHub account is not required to simply clone the repo.

**Review by QYAN24 — APPROVED** [review]
Looks good, see one minor comment before merging.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+ 
+### README.md
+This is what you are reading now. It contains everything you need to know to understand the contents of the repository and how to use it.
+  
+### Bus Stop Census.Rproj
+This is the R project file for the Bus Stop Census analysis. An R Project is simply a working directory designated with a .RProj file. When you open a project (using File/Open Project in RStudio or by double–clicking on the .Rproj file outside of R), the working directory will automatically be set to the directory that the .RProj file is located in.
+
+### .RProfile
+This is a script that runs every time R starts. The `.RProfile` included in this repository will run each time the `Bus Stop Census.RProj` file is opened. The project `.RProfile` script is important because it installs and loads the libraries needed to run the code as well as the GTFS data that is the key to linking bus stop IDs with their associated route, coordinate, and schedule information.
+
```
Comment (QYAN24) [inline README.md]: Does RStudio automatically install any required packages or do they need to be manually installed first? If a manual installation of packages is necessary, please provide instructions for that.

File: README.md (hunk 1 above)
Comment (morganjacobus) [reply README.md]: I prefer the second. I will update it

File: README.md (hunk 2 above)
Comment (morganjacobus) [reply README.md]: Agreed. Removing

File: README.md (hunk 3 above)
Comment (morganjacobus) [reply README.md]: I will update the .RProfile to both install and load the necessary packages. By doing this, the users will not have to fiddle with manually installing them

**Review by QYAN24 — APPROVED** [review]
Thanks for the changes.

**Pull request merged**